### ISSUE TITLE
Varovalke

### ISSUE THREAD

**Issue by golobluka** [golobluka/sticker#1]
Nisva še dogovorila problema zagrajevanja programa, kar onemogoča neprijerne erorje in opozorila. V prvi izdaji sta funkciji available_row in available_number, ki onemogočata da bi igralec vzel več kart kot jih v vrsti sploh je. Ker pa bo vmesnik sedaj drugačen rabiva tudi drugačne varovalke.

**Comment by HyoMiYing** [comment]
Ja, to je dobra poanta. Najprej bi predlagal, da poimenujeva te funkcije 'validacijske_funkcije', celemu pojmu pa rečeva 'validacija' - tako je jasneje za večino ljudi.

**Comment by HyoMiYing** [comment]
Tako bi imeni funkcij izgledali sledeče:
`def validate_row(self, row):` ali pa npr. `def row_validation(self, row):`
ter
`def validate_number(self, row, number):` ali pa npr. `def number_validation(self, row, num):`

**Comment by HyoMiYing** [comment]
Funkcije bi pa spremenil na tak način, da namesto print() stavka vrnejo Error string.

**Comment by HyoMiYing** [comment]
Funkcija [Try/Except](https://www.w3schools.com/python/python_try_except.asp) je precej primerna za tak primer.

Najbolje je, da ob uspešni validaciji funkcija vrne npr.: "Row is valid", ob neuspešni pa "Row invalid"

**Comment by golobluka** [comment]
Ok, ja se strinjam glede pojmovanja. Ko bos na zadostni stopnji napisi se
konkretno katere funkcije bi potreboval, se bom pa se prijavil na skayp da
bova bol rekoce komunicirala.
